replacer.ts

```diff
@@ -38,25 +40,23 @@ export class LinkReplacer {
 
     let result = content;
     for (const [idx, link] of links.entries()) {
-      status.setText(
-        `Wayback: Replacing ${idx + 1} of ${links.length} link(s)...`,
-      );
+      status.setText(STATUS_MESSAGES.REPLACING(idx + 1, links.length));
 
       const url = new URL(link);
-      if (HOSTNAMES_BLOCKED.includes(url.hostname)) {
-        console.warn(`Ignoring blocked hostname: ${url.hostname}`);
+      if (HOSTNAMES_BLOCKED.includes(url.hostname as typeof HOSTNAMES_BLOCKED[number])) {
```

Comment:
I don't think this is an idiomatic way? We should not need to cast the type should we?